appendix/basic_commands.tex

```diff
@@ -1545,10 +1545,11 @@ \subsection{GOSUB}
 }
 
 The above program will print {\ttfamily FIRST}, {\ttfamily SECOND}, and
-{\ttfamily THRID} in order.  This is because the {\ttfamily GOSUB} statement on
+{\ttfamily THIRD} in order.  This is because the {\ttfamily GOSUB} statement on
 line 20 transfers execution to line 50, and then the {\ttfamily RETURN}
-statement on line 60 then transfers execution back to line 30, which is the
-line {\em after} the {\ttfamily GOSUB} statement was called.\\
+statement on line 60 transfers execution back to line 30, which is the
+line {\em after} the {\ttfamily GOSUB} statement was called. Line 40 ends the 
```

Comment:
Not sure if I have this mentioned anywhere, but use 2 spaces between sentences...as God intended!

This is not just some arbitrary holy war, however.  I have some commands set up in my editor that look for two spaces as a delimiter for sentences.  I use them for editing certain things.  LaTeX will determine the space between sentences no matter how many you use, but I don't want my delimiters going away.

Reply:
Well call them "standards" for a reason. You don't have to agree with them; you just have to follow them.  ;-)

Reply:
Ok.  2 spaces between sentences.  👌